app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/components/ClustersStepForm/ClustersStepForm.js

```diff
@@ -142,7 +147,11 @@ class ClustersStepForm extends React.Component {
           {targetClusters && (
             <DualPaneMapperList
               id="target_clusters"
-              listTitle={__('Target Clusters')}
+              listTitle={
+                selectedTargetCluster && selectedTargetCluster.ext_management_system.emstype !== 'rhevm'
```

Comment:
There is going to be a `targetProvider` key in `redux-form` that will be set by the dropdown in Step 1 of the mapping wizard.  (see [Step 1 PR](https://user-images.githubusercontent.com/15141412/43005535-2d8c33c8-8c01-11e8-833a-6bb98c4c0975.png))

Not sure if there are plans to add other providers other than RHV and OSP, but in that case, it would be nice to have a mapping in one of the constants files like

```js
const providerMapping = {
  osp: __('Target Provider \\ Project'),
  rhv: __('Target Provider \\ Datacenter \\ Cluster')
}
```
and use the redux data to choose the correct string.

Reply:
Ok cool, I've updated my PR so that the value stored in `redux` will be `rhevm`

Reply:
hah now all we need is a better way of knowing which pr depends on which pr 😆

Reply:
😆 I know, I think while we work on these issues, I might keep a local branch where I rebase all the PRs together for testing/review as we go along.  Then, when everything looks good, we can merge the individual PRs one by one.  What do you think?